components/admin_console/group_settings/__snapshots__/group_settings.test.jsx.snap

```diff
@@ -21,7 +21,7 @@ exports[`components/admin_console/group_settings/GroupSettings should match snap
     >
       <InjectIntl(FormattedMarkdownMessage)
         defaultMessage="Groups are a way to organize users and apply actions to all users within that group.
-For more information on Groups, please see [documentation](!https://about.mattermost.com)."
+For more information on Groups, please see [documentation](!http://www.mattermost.com/default-ad-ldap-groups)."
```

Comment:
should we default to `https` or `http` for website links?

Reply:
^ @wiersgallak

Reply:
Always HTTPS, please